coderadar-test/src/main/asciidoc/restapi.adoc

```diff
@@ -85,10 +88,8 @@ To access the functionality of coderadar you have to register a user.
 You need to define a username and a password. The password will be sent as
 plain text and hashed on server side for persisting.
 
-==== Hypermedia links
-include::{snippets}/user/registration/links.adoc[]
-
 ==== Registering a User
+// TODO should password really be send as plaintext or is it in plaintext but encrypted?
```

Comment:
The password is sent as plaintext over HTTPS, so yes, in a way, everything we send to the server should be encrypted.